### PR TITLE
cve: Suppress false +ves

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: cve: Bump curl version and suppress false +ves
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: ci/do_ci.sh [hunk 1]
```diff
@@ -452,6 +452,9 @@ elif [[ "$CI_TARGET" == "docs" ]]; then
   exit 0
 elif [[ "$CI_TARGET" == "deps" ]]; then
 
+  # TEMP: run cve scan
```
Comment (phlax) [inline ci/do_ci.sh:455]: remove

**Comment by phlax** [comment]
@htuch @moderation not sure if we want to bump curl in this PR given that it doesnt fix the cve warning

**Comment by htuch** [comment]
@phlax in that case probably just make this a cve-scan PR and we can merge.

**Comment by phlax** [comment]
ill sieve out the change...

**Review by htuch — APPROVED** [review]

**Pull request merged**